web/themes/custom/pineapple/templates/user/user--account--full.html.twig

```diff
@@ -18,28 +18,83 @@
           {% include "@components/_heading.twig" with {
             heading_level: 1,
-            heading_attributes: create_attribute().addClass('author-name'),
+            heading_attributes: create_attribute().addClass('subheading'),
             heading_text: content.field_u_name|field_value|first['#context'].value
           } %}
+        </div> <!-- End of author-name-container -->
+        <div class="author-info">
+          {% set user_roles = user.getRoles() %}
+          {% set member_since = user.getCreatedTime()|date("d/m/Y") %}
+          <div class="member-since">
+            <strong>Member since</strong>
+            <p>{{ member_since }}</p>
+          </div>
+          {% if 'recipe_editor' in user_roles or 'article_editor' in user_roles %}
+            <div class="roles">
+                <strong>Role(s)</strong>
+              {% if 'recipe_editor' in user_roles %}
+                <p>Recipe Editor</p>
+              {% endif %}
+              {% if 'article_editor' in user_roles %}
+                <p>Article Editor</p>
+              {% endif %}
+            </div> <!-- End of roles -->
+          {% endif %}
+        </div> <!-- End of author-info -->
+        {% if content.field_u_biography|field_value %}
           <p>{{ content.field_u_biography|field_value }}</p>
-        </div> <!-- End of author-text -->
-      </div> <!-- End of author-info-container -->
-    </div> <!-- End of author-container -->
+        {% endif %}
+        {% if 'recipe_editor' in user_roles or 'article_editor' in user_roles %}
+          <div class="sidebar-buttons u-o-1">
+            {% set current_user_id = bamboo_load_currentuser().id %}
+            {% if current_user_id == user.id %}
+              {% set edit_link = '/user/' ~ user.id ~ '/edit'  %}
+              {% include "@components/_link.twig" with {
+                link_text: 'Edit Profile',
+                link_url: edit_link,
+                link_attributes: create_attribute().addClass('btn', 'btn--primary')
+              } %}
+              {% if 'recipe_editor' in user_roles or 'article_editor' in user_roles %}
+                {% include "@components/_link.twig" with {
+                  link_text: 'Create Content',
+                  link_url: '/node/add',
+                  link_attributes: create_attribute().addClass('btn', 'btn--tertiary')
+                } %}
+              {% endif %}
+            {% endif %}
+          </div>
+        {% endif %}
+      </div> <!-- End of author-container -->
+      <div class="sidebar-links">
+        {% include "@components/_link.twig" with {
+          link_text: 'How To Create Content',
+          link_url: '/help'
+        } %}
+        {% include "@components/_link.twig" with {
+          link_text: 'Contact Us',
+          link_url: '/contact-us'
+        } %}
+        {% include "@components/_link.twig" with {
+          link_text: 'Privacy Policy',
+          link_url: '/privacy-policy'
+        } %}
+      </div>
```

Comment:
Do you think it's worth adding a heading to this column, something like "Useful" or "Helpful" ~ "links"? it would give it more context at first glance rather than reading all of them...

Reply:
I didn't think it was necessary but I like how it looks so ... done.